sunpy/coordinates/spice.py

```diff
@@ -467,3 +467,65 @@ def get_fov(instrument, time, *, resolution=100):
                    obstime=obstime,
                    representation_type='unitspherical')
     return fov
+
+def get_rotation_matrix(source_frame, target_frame, from_time, to_time=None):
+    """
+    Computes the rotation matrix that converts vectors from the `source_frame` to the `target_frame`
+    at the specified time using the SPICE toolkit.
+
+    Parameters
+    ----------
+    source_frame : `str`, `int`, `~astropy.coordinates.BaseCoordinateFrame`
+        The frame of the input vector field, specified as a frame name, SPICE ID, or Astropy BaseCoordinateFrame.
+
+    target_frame : `str`, `int`, `~astropy.coordinates.BaseCoordinateFrame`
+        The frame to which the vector field will be transformed, specified in the same formats as ``source_frame``.
+
+    from_time : `str`, `~astropy.time.Time`, `~datetime.datetime`, `~datetime.date`, `~numpy.datetime64`, `~pandas.Series`, `~pandas.DatetimeIndex`, `~pandas.DataFrame`
```

Comment:
We have a decorator to add the time formats in our docstrings. I forget whats it called.

Reply:
Yeah got it.